ArtsofteDAL/Models/Department.cs

```diff
@@ -1,19 +1,15 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Text.Json.Serialization;
 
-namespace ArtsofteDAL.POCO_Entities
+namespace ArtsofteDAL.Models
 {
     [SuppressMessage("ReSharper", "InconsistentNaming")]
     public class Department
     {
-        public int DepartmentID { get; set; } 
-        public int EmployeeID { get; set; }
```

Comment:
Why this is only ref to employee and not the whole employee, what is the benefit of that?